docs/resources/modelarts_dataset.md

```diff
@@ -0,0 +1,195 @@
+  
+ Changing this parameter will create a new resource.
+
+* `path` - (Optional, String, ForceNew) Specifies the OBS path when `data_type` is `0`
+ or the hdsf path when `data_type` is `4`. All the file in this directory and subdirectories will be which be imported
+ to the dataset. Changing this parameter will create a new resource.
+
+* `with_column_header` - (Optional, Bool, ForceNew) Specifies whether the data contains table header when the type
+ of dataset is `400`(Table type). Default value is `true`. Changing this parameter will create a new resource.
+
+* `queue_name` - (Optional, String, ForceNew) Specifies the queue name of DLI when `data_type` is `2`.
+ Changing this parameter will create a new resource.
+
+* `database_name` - (Optional, String, ForceNew) Specifies the database name of DWS/DLI when `data_type` is `1` or `2`.
+ Changing this parameter will create a new resource.
+
+* `table_name` - (Optional, String, ForceNew) Specifies the table name of DWS/DLI when `data_type` is `1` or `2`.
+ Changing this parameter will create a new resource.
+
+* `cluster_id` - (Optional, String, ForceNew) Specifies the cluster ID of DWS/MRS when `data_type` is `1` or `4`.
+ Changing this parameter will create a new resource.
+
+* `user_name` - (Optional, String, ForceNew) Specifies the user name of databse when `data_type` is `1`.
+ Changing this parameter will create a new resource.
+
+* `password` - (Optional, String, ForceNew) Specifies the password of database when `data_type` is `1`.
+ Changing this parameter will create a new resource.
+
+The `schemas` block supports:
+
+* `type` - (Required, String, ForceNew) Specifies the field type. Valid values include: `String`, `Short`, `Int`,
+ `Long`, `Double`, `Float`, `Byte`, `Date`, `Timestamp`, `Bool`. Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the field name. Changing this parameter will create a new resource.
+
+The `label_format` block supports:
+
+* `type` - (Required, Int, ForceNew) Specifies Label type for text classification.
+ The optional values are as follows:
+
+  + **0**: Label and text are separated, distinguished by the suffix `_result`.
+   For example: the text file is `abc.txt`, and the label file is "`abc_result.txt`.
+  + **1**: Default, labels and text are in one file, separated by a delimiter. The separator between text and labels,
+   the separator between label and label can be specified by `label_separato` and `text_label_separator`.
+
+* `text_label_separator` - (Optional, String, ForceNew) Specifies the separator between text and label.
+ Changing this parameter will create a new resource.
+
+* `label_separator` - (Optional, String, ForceNew) Specifies the separator between label and label.
+ Changing this parameter will create a new resource.
+
+The `labels` block supports:
+
+* `name` - (Required, String, ForceNew) Specifies the name of label. Changing this parameter will create a new resource.
+
```

Comment:
The parameter `type` is missing.

Reply:
The type is unnecessary in console, so remove from schema